routes/api.php

```diff
@@ -14,6 +14,6 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
+Route::middleware(['client'])->group(function () {
+    Route::get('validate/{key}', [ValidationController::class, 'index'])->name('validation.index');
```

Comment:
The initial route was `'{secretKey}/validate/{key}'`. Do you get rid of the `secretKey`?

Reply:
Why would we need it?

Reply:
When using passport, the webcall will be authenticated. It's not necessary to protect the url more.